extensions/applicationinsights-debugplugin-js/src/DebugPlugin.ts

```diff
@@ -327,7 +335,11 @@ export default class DebugPlugin extends BaseTelemetryPlugin {
                     }
 
                     let evtPrefix = _getEvtPrefix(funcArgs);
-                    dashboard.newLogEntry(_createInstrumentObject(funcArgs, orgArgs), dateNow() - startTime, `${evtPrefix}`, 0, funcArgs.name);
+                    let obj = _createInstrumentObject(funcArgs, orgArgs);
+                    dashboard.newLogEntry(obj, dateNow() - startTime, evtPrefix, 0, funcArgs.name);
+
+                    _debugExtMsg(evtPrefix, obj);
+
```

Comment:
Repeated several times, can be extracted into a function?

Reply:
Done (again 😄 )